sdk/src/main/java/software/amazon/awssdk/iot/discovery/DiscoveryClientConfig.java

```diff
@@ -164,6 +169,27 @@ public String getGGServerName() {
         return ggServerName;
     }
 
+    /**
+     * @return the executor set for this discover client, if one is set.
```

Comment:
Semantics but it returns whether an executor is set or not. This should probably read that it returns the set executor or null.